.env.example

```
@@ -111,6 +111,44 @@ BREVO_LIST_ID=
+# Twilio (for international phone numbers via Verify API)
+# Get from: Twilio Console → Account Dashboard
+TWILIO_ACCOUNT_SID=your_twilio_account_sid
+TWILIO_AUTH_TOKEN=your_twilio_auth_token
+# Verify service SID: Twilio Console → Verify → Services → create or use default
+TWILIO_VERIFY_SERVICE_SID=VAxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# =============================================================================
+# SmileID KYC Verification Services
+# =============================================================================
+
+SMILE_IDENTITY_BASE_URL=XXXXXX #get from docs
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)